docs/user-guide/transformer_expressions.md

```diff
@@ -535,3 +549,397 @@ variables:
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: min
+  expression: yourTableName.Col("avg_order_1_day").Min()
+```
+Output: 2000
+
+### MinStr
+
+`MinStr` is a function to find minimum value from column values. The output will be single value in string type
+
+#### Input
+No Input
+
+#### Output
+Single value with string type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: min_str
+  expression: yourTableName.Col("avg_order_1_day").MinStr()
+```
+Output: "2000"
+
+### Quantile
+
+`Quantile` is a function to returns the sample of x such that x is greater than or equal to the fraction p of samples
+
+#### Input
+Fraction in float type
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| rank |
+|-------|
+| 1 |
+| 2 | 
+| 3 |
+| 4 |
+| 5 |
+| 6 | 
+| 7 |
+| 8 |
+| 9 |
+| 10 |
+
+Standard Transformer Config:
+```
+variables:
+- name: quantile_0.9
+  expression: yourTableName.Col("rank").Quantile(0.9)
+```
+Output: 9
+
+### Sum
+
+`Sum` is a function to sum all the values in the columns. The output will be single value in float type
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: sum
+  expression: yourTableName.Col("avg_order_1_day").Sum()
+```
+Output: 9000
+
+### Flatten
+`Flatten` is a function to flatten all values in a column, this is suitable for column that has list type, for non list the result will be the same with the original columns
+
+#### Input
+No Input
+
+#### Output
+New Column that the value already flatten
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | nearby_restaurant_ids |
+|---------------|-----------------|
+| 1 | [2, 3, 4] |
+| 2 | [4, 5, 6] |
+| 3 | [7, 8, 9] |
+
+Standard Transformer Config:
+```
+variables:
+- name: restaurant_ids
+  expression: yourTableName.Col("nearby_restaurant_ids").Flatten()
+```
+Output:
+| restaurant_ids |
+|----------------|
+| 2 |
+| 3 |
+| 4 |
+| 4 |
+| 5 |
+| 6 |
+| 7 |
+| 8 |
+| 9 |
+
+### Unique
+`Unique` is a function to return all values without duplication. 
+#### Input
+No Input
+
+#### Output
+New Column that has unique value for each row
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | rating |
+|---------------|-----------------|
+| 1 | [2, 3, 4] |
```

Comment:
What happen to the second sub-column of `restaurant_id` ?

Reply:
what do you mean by sub-column of `restaurant_id` ? do you mean what happen if `Unique` is run for `rating` column?

Reply:
nvm, I thought `restaurant_id` has a nested column since I am only reading the raw markdown. 
Follow up question, what is the output of `yourTableName.Col("rating").Unique()` expression?

Reply:
I already updated in the docs